MAUI/NumericEntry/Getting-Started.md

```diff
@@ -0,0 +1,266 @@
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry value editing](GettingStarted_images/editing_value.gif)
+
+## Change number format
+
+You can change the format in which the value should be displayed using the `CustomFormat` property . By default, value of the `CustomFormat` property is **null**. 
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumericEntry HorizontalOptions="Center"
+                     VerticalOptions="Center"
+                     Value="12.5" 
+                     CustomFormat="C2" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumberBox sfNumberBox = new SfNumberBox();
+sfNumberBox.HorizontalOptions = LayoutOptions.Center;
+sfNumberBox.VerticalOptions = LayoutOptions.Center;
+sfNumberBox.Value = 12.5;
+sfNumberBox.CustomFormat = "C2";
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry value editing](GettingStarted_images/custom_format.png)
+
+## Accept null value
+
+By default, `NumericEntry` control allows **null** value. A null value is assigned when the user clicks the clear button or clears the input. You can disable this by setting the value of `AllowNull` property as **false**. When value of the `AllowNull` property is set to **false** and the input is cleared, the `NumericEntry` control returns it to **0**. 
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumericEntry HorizontalOptions="Center"
+                     VerticalOptions="Center"
+                     Value="12315"
+                     AllowNull="False" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericEntry sfNumericEntry = new SfNumericEntry();
+sfNumericEntry.HorizontalOptions = LayoutOptions.Center;
+sfNumericEntry.VerticalOptions = LayoutOptions.Center;
+sfNumericEntry.AllowNull = false;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry prevent empty textbox](GettingStarted_images/allowNull.gif)
+
+## Setting Placeholder Text
+
+You can prompt the user with any information by using the `PlaceholderText` property. Watermark text will be displayed only when the value of the `AllowNull` property is **true** and the value of `NumericEntry` control is **null**. The default value of `Placeholder` property is **string.Empty** (No string will be displayed).
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumberBox HorizontalOptions="Center" 
+                     VerticalOptions="Center" 
+                     Placeholder="Enter input here..." />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumberBox SfNumberBox= new SfNumberBox();
+SfNumberBox.Placeholder = "Enter input here...";
+sfNumberBox.HorizontalOptions = LayoutOptions.Center;
+sfNumberBox.VerticalOptions = LayoutOptions.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry Placeholder Text](GettingStarted_images/placeholder_text.png)
+
+## Clear button visibility 
+
+The `ShowClearButton` property is used to show or hide the clear button in `NumericEntry`. By default, visibility of the clear button is enabled.
```

Comment:
The `ShowClearButton` property is used to show or hide the clear button in the `NumericEntry`. By default, the visibility of the clear button is enabled.

Reply:
The requested content modification has been successfully addressed